### DISCUSSION TITLE
Marker/Trail Pack for Legendary Collections

### DISCUSSION THREAD

**Discussion by wolfmurmur** [blish-hud/Blish-HUD#719] [Requested Marker Packs]
Hoping to see some packs that help to point out where exactly one needs to go for certain collection items, e.g. Envoy I's Earth Elemental in Tangled Depths, or the various spots in Auric Basin :)

**Comment by Metallis** [comment]
Some of these can be challenging, but I can give it a shot. Can you get a full list of things you'd like to see included? Would help me avoid scope creep, etc.

**Reply by wolfmurmur** [reply]
Sure! I was just thinking about stuff like the different little collection things you need to do for the Envoy (1 and 2) collections, e.g. "Obtain a Soul Mirror from Burnisher Kengo" or the Giant Beehive JP in Salvation Pass. Another good example would be the Ad Infinitum collection and the relevant motes + dungeon mob drops. Right now, I've only been able to find written guides on these objectives, which while helpful, aren't quite the same as an in-game visual aid and are in some ways lacking. I found myself wandering aimlessly like "It says it should be right around here..."

I'm unfamiliar with other legendary collections, but I imagine they also have similar "find these random NPCs you barely interact with", "do this puzzle to get collection item" etc objectives where a visual aid would also be helpful.
